example/src/InputWithOptions.js

```diff
@@ -1,39 +1,39 @@
-import React, { useState } from "react";
-import { useStateValidator } from "react-indicative-hooks";
+import React, { Fragment, useState } from 'react';
+import { useStateValidator } from 'react-indicative-hooks';
 
-const rules = "required";
+const rules = 'required';
 
 const messages = {
-  required: "Please, fill the input with some data"
+    required: 'Please, fill the input with some data'
 };
 
 export default function InputWithOptions() {
-  const [show, setShow] = useState(false);
+    const [show, setShow] = useState(false);
 
-  const options = {
-    messages,
-    runOnMount: true,
-    onSuccess: () => setShow(false),
-    onError: () => setShow(true)
-  };
+    const options = {
+        messages,
+        runOnMount: true,
+        onSuccess: () => setShow(false),
+        onError: () => setShow(true)
+    };
 
-  const [value, setValue, error] = useStateValidator("", rules, options);
+    const [value, setValue, error] = useStateValidator('', rules, options);
 
-  return (
-    <>
-      <p className="text-red mb-5" data-testid="alert">
-        {show && "Sorry, you have an error"}
-      </p>
-      Value
-      <input
-        value={value}
-        onChange={e => setValue(e.target.value)}
-        data-testid="input"
-        className="ml-2 mr-5 mb-5"
-      />
-      <p className="text-red" data-testid="error">
-        {error && error.message}
-      </p>
-    </>
-  );
+    return (
+        <Fragment>
```

Comment:
Why you don't like the sugar for <Fragment>? hahaha
For me, it cut one import and don't write unnecessary things in the middle of your jsx.

Reply:
I just don't like to have `<>` because some times you need `<Fragment key="SOME_KEY">` because consistency. And don't like to have `<React.Fragment>` because otherwise you can even have `<Something.SomethingElse.SomeComponent>` which makes everything even hard to read.

But I don't see a big deal if you like it the other way. it's your package. :)

Tell me if I should change it back.

Reply:
I don't even import them by hand nowadays, my IDE does it for me automatically. ;)

Reply:
Haha, no problem to me at all, it's not a blocking change!

I told you, just because it's annoying reading non rendered tags, and even removing the Fragment import when you don't need it anymore (import is automatically, but delete not, hahaha).

But, no problem to me, you can keep it like this. :)